app/views/components/pageflow/admin/add_membership_button_if_needed.rb

```diff
@@ -4,39 +4,45 @@ class AddMembershipButtonIfNeeded < ViewComponent
       builder_method :add_membership_button_if_needed
 
       def build(user, parent, entity_type)
-        if parent.is_a?(User)
-          button_label = I18n.t("pageflow.admin.#{entity_type}.add")
-          path = new_admin_user_membership_path(
-            parent, entity_type: to_class_name(entity_type)
-          )
-          data_tooltip = I18n.t("pageflow.admin.#{entity_type}.none_addable_tooltip")
-          rel = "add_#{entity_type}_membership"
-        elsif parent.is_a?(Entry)
-          path = new_admin_entry_membership_path(
-            parent, entity_type: to_class_name(entity_type)
-          )
+        if !Pageflow.config.allow_multiaccount_users &&
```

Comment:
This sort of duplicated permission logic in the view layer. We could try to use `authorize?(:create, membership)` with an unsaved example membership instead. The logic to construct that example membership would be closely related to what is already used below to find out if another membership would be possible. Then we could also remove the `authorize?` call in the  [members tab](https://github.com/codevise/pageflow/blob/b6610062f7a37bcdd0463cfef2f5159c6a0ae219/app/views/components/pageflow/admin/members_tab.rb#L49). 

Maybe take stab at moving in that direction. But if it turns out to take more than one or two hours, I'm not sure it's worth it.

Reply:
As discussed, I'll go in the direction of putting another `authorize?` call into UserAccountsTab and rename `AddMembershipButtonIfNeeded` to `AddMembershipButton`

Reply:
And `UsersTab` which is displayed in the account admin show page.

Reply:
I only saw this last comment now while reading my e-mail. Looking into it